esx_service/utils/vmdk_utils.py

```diff
@@ -494,6 +495,14 @@ def get_datastore_url_from_config_path(config_path):
                   % (config_path, config_ds_url))
     return config_ds_url
 
+def get_version():
+    """ Return the version of the installed VIB """
+    try:
+        cmd = 'localcli software vib list | grep esx-vmdkops-service'
```

Comment:
VIB name could be a constant defined earlier.

Reply:
Done